x-pack/plugins/observability/public/pages/overview/old_overview_page.tsx

```diff
@@ -263,11 +270,19 @@ export function OverviewPage({ routeParams }: Props) {
 }
 
 interface PageHeaderProps {
+  showTour?: boolean;
   handleGuidedSetupClick: () => void;
   onTimeRangeRefresh: () => void;
 }
 
-function PageHeader({ handleGuidedSetupClick, onTimeRangeRefresh }: PageHeaderProps) {
+function PageHeader({
+  showTour = false,
+  handleGuidedSetupClick,
+  onTimeRangeRefresh,
+}: PageHeaderProps) {
+  const [isTourVisible, setIsTourVisible] = useState(true);
```

Comment:
Is this state necessary? Could we pass a function to modify the state in the parent? I think it would make the code simpler